addons/l10n_id/data/account_chart_template_post_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="l10n_id" model="account.chart.template">
+        <field name="property_account_receivable_id" ref="a_1_121001"/>
+        <field name="property_account_payable_id" ref="a_2_110001"/>
+        <field name="property_account_expense_categ_id" ref="a_5_100001"/>
+        <field name="property_account_income_categ_id" ref="a_4_100001"/>
+        <field name="income_currency_exchange_account_id" ref="a_8_110001"/>
+        <field name="expense_currency_exchange_account_id" ref="a_9_110001"/>
+        <field name="use_anglo_saxon" eval="1"/>
```

Comment:
Stock input/output accounts?

Reply:
@jco-odoo any suggestions which accounts to be used as default stock input/output/valuation accounts ?

Reply:
The stock interim is: 
"a_2_900000","2900000"

Usually we use this for input and output.

The inventory is split into several acocunts from 

"a_1_130001","1130001"
To
"a_1_130018","1130018"

But we can use the latest for the default value